graphics/fontforge/Portfile

```diff
@@ -90,66 +46,54 @@ depends_lib         port:freetype \
                     port:libuninameslist \
                     port:libxml2 \
                     port:tiff \
-                    port:xorg-libXi \
-                    port:xorg-libXt \
-                    port:xorg-libxkbui \
-                    port:zlib
-
-use_autoconf        yes
+                    port:zlib \
+                    port:libspiro \
+                    port:pango \
+                    port:cairo \
+                    port:potrace
 
 configure.args      --mandir=${prefix}/share/man \
-                    --enable-type3 \
-                    --enable-devicetables \
-                    --enable-gb12345 \
-                    --without-python \
-                    --without-freetype-bytecode \
-                    --without-pango \
-                    --without-cairo \
-                    --x-includes=${prefix}/include \
-                    --x-libraries=${prefix}/lib
+                    --disable-python-scripting \
+                    --disable-python-extension \
+                    --without-libzmq \
+                    --without-x
 
 configure.ldflags-append \
                     -lintl
 
-# https://trac.macports.org/ticket/29840
-compiler.blacklist  llvm-gcc-4.2
-
-set docdir          ${prefix}/share/doc/${name}
-destroot.args       docdir=${docdir}
-
-post-destroot {
-    xinstall -d ${destroot}${docdir}
-    xinstall -m 644 -W ${worksrcpath} \
-        AUTHORS INSTALL LICENSE README-Mac.html README-Unix.html README-unix \
-        ${destroot}${docdir}
-    # Fix permissions problem on this file
-    xinstall -m 0644 -W ${destroot}${applications_dir}/FontForge.app/Contents/Resources FontForge.icns \
-        ${destroot}${applications_dir}/FontForge.app/Contents/Resources
+variant python27 description {Enable Python support (Python 2.7)} {
+    depends_lib-append      port:python27
+    configure.args-delete   --disable-python-scripting \
+                            --disable-python-extension
+    configure.python        ${prefix}/bin/python2.7
+    configure.pkg_config_path \
+                            "${frameworks_dir}/Python.framework/Versions/2.7/lib/pkgconfig"
 }
 
-variant freetype_bytecode description {Enable support for bytecode interpreter} {
-    distfiles-append        freetype-${ftversion}${extract.suffix}:freetype
-    extract.only-append     freetype-${ftversion}${extract.suffix}
-    configure.args-delete   --without-freetype-bytecode
-    configure.args-append   --with-freetype-src=${workpath}/freetype-${ftversion}
+variant gui description {Enable GUI support} {
+    depends_lib-append      port:xorg-libXi
```

Comment:
Isn't also `py27-pygtk` missing here? I didn't test since months though, so I'm not sure.

Reply:
Hmm, I'm not sure. AFAIK you don't need `py27-pygtk` for the UI. At least not in this version. They are planning to switch from X to gtk in the future, but this hasn't happened in this version yet.

Reply:
Ignore it then. It could be that I tested master branch. Feel free to merge.

Reply:
I'll merge tomorrow if there are no other discussion points.

Reply:
(Or possibly in about four hours depending on my schedule.)